docs/integrations/destinations/duckdb.md

```diff
@@ -106,7 +106,8 @@ Note: If you are running Airbyte on Windows with Docker backed by WSL2, you have
 
 | Version | Date       | Pull Request                                             | Subject                |
 | :------ | :--------- | :------------------------------------------------------- | :--------------------- |
-| 0.3.3   | 2024-0407 | [#36884](https://github.com/airbytehq/airbyte/pull/36884) | Fix stale dependency versions in lock file, add CLI for internal testing.  |
+| 0.3.4   | 2024-04-08 | [#36715](https://github.com/airbytehq/airbyte/pull/36715) | Improve ingestion performance using pyarrow inmem view for writing to DuckDB.  |
```

Comment:
Estimating release tomorrow. 😄🚀 

```suggestion
| 0.3.4   | 2024-04-16 | [#36715](https://github.com/airbytehq/airbyte/pull/36715) | Improve ingestion performance using pyarrow inmem view for writing to DuckDB.  |
```

We can update again if needed. (Also, doesn't have to be exactly correct.)